embedding/csharp/Tizen.Flutter.Embedding/ElmFlutterView.cs

```diff
@@ -0,0 +1,193 @@
+﻿// Copyright 2022 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+using System;
+using System.Diagnostics;
+using ElmSharp;
+using static Tizen.Flutter.Embedding.Interop;
+
+namespace Tizen.Flutter.Embedding
+{
+    /// <summary>
+    /// Represents an <see cref="EvasObject"/> instance created by the embedder.
+    /// </summary>
+    class EvasObjectImpl : EvasObject
+    {
+        public EvasObjectImpl(EvasObject parent, IntPtr handle) : base(parent)
+        {
+            Handle = handle;
+        }
+
+        protected override IntPtr CreateHandle(EvasObject parent)
+        {
+            return Handle;
+        }
+    }
+
+    /// <summary>
+    /// Displays a Flutter screen in a Tizen application.
+    /// </summary>
+    public class ElmFlutterView : IPluginRegistry
+    {
+        /// <summary>
+        /// The Flutter engine instance.
+        /// </summary>
+        private FlutterEngine Engine { get; set; } = null;
```

Comment:
How about changing Engine getter to public? I think we need to deliver the Pause, Resume event to Engine.

Reply:
I just followed https://github.com/flutter-tizen/flutter-tizen/blob/8d6f09c254575b3e66b7d070e78adaec9fa18ed3/embedding/cpp/include/elm_flutter_view.h#L46-L48

and notifying the engine of any event will require a dedicated API or something. (You might refer to https://github.com/flutter-tizen/flutter-tizen/pull/378.)

We can change the access modifier to public (or protected) any time later if we need.

Reply:
I had some concerns, so I left [it](https://github.com/flutter-tizen/flutter-tizen/pull/378) as draft.
One of the concerns was to provide the engine reference of the view publicly instead of view's methods for life-cycle. 

Which one looks better(provide an engine reference as public or methods for life-cycle)?